src/tests/engines_util/execute_tools.hpp

```diff
@@ -4,51 +4,33 @@
 
 #pragma once
 
-#include <exception>
-#include <fstream>
-#include <iomanip>
-#include <iostream>
-#include <list>
-#include <memory>
 #include <random>
-#include <vector>
 
 #include "gtest/gtest.h"
-#include "ngraph/file_util.hpp"
-#include "ngraph/function.hpp"
-#include "ngraph/log.hpp"
-#include "ngraph/node.hpp"
-#include "ngraph/op/op.hpp"
 #include "ngraph/runtime/host_tensor.hpp"
-#include "ngraph/runtime/tensor.hpp"
-#include "ngraph/type/element_type_traits.hpp"
+#include "openvino/core/model.hpp"
+#include "openvino/core/node.hpp"
+#include "openvino/core/shape.hpp"
+#include "openvino/op/op.hpp"
 
-namespace ngraph {
+namespace ov {
 class TestOpMultiOut : public op::Op {
```

Comment:
I think we can remove this `TestOpMultiOut` from the library and move to core tests... 
Looks like we don't use this operation in other places

Reply:
done